charms/reactive/bus.py

```diff
@@ -443,12 +445,16 @@ def _load_module(root, filepath):
 
 
 def _register_handlers_from_file(root, filepath):
+    exec_whitelist = ('', '.py', '.sh')
     no_exec_blacklist = (
-        '.md', '.yaml', '.txt', '.ini',
         'makefile', '.gitignore',
-        'copyright', 'license')
+        'copyright', 'license',)
```

Comment:
Autoformatting change? Seems unnecessary to just add a comma

Reply:
Copy-pasta.  I'll remove.